static/app/components/replays/table/replayTable.tsx

```diff
@@ -114,37 +100,13 @@ export default function ReplayTable({
   );
 }
 
-type TableProps = {
-  children: ReactNode;
+const ReplayTableWithColumns = styled(SimpleTable, {
+  shouldForwardProp: prop => prop !== 'columns',
+})<{
   columns: readonly ReplayTableColumn[];
-  onSortClick?: (key: string) => void;
-  sort?: Sort;
-} & HTMLAttributes<HTMLTableElement>;
-
-const ReplayTableWithColumns = styled(
-  ({children, columns, onSortClick, sort, ...props}: TableProps) => (
-    <SimpleTable {...props}>
-      <SimpleTable.Header>
-        {columns.map((column, columnIndex) => (
-          <SimpleTable.HeaderCell
-            key={`${column.name}-${columnIndex}`}
-            handleSortClick={() => column.sortKey && onSortClick?.(column.sortKey)}
-            sort={
-              column.sortKey && sort?.field === column.sortKey ? sort.kind : undefined
-            }
-          >
-            <Tooltip title={column.tooltip} disabled={!column.tooltip}>
-              {column.name}
-            </Tooltip>
-          </SimpleTable.HeaderCell>
-        ))}
-      </SimpleTable.Header>
```

Comment:
This moved into `replayTableHeader.tsx` where we conditionally switch to show bulk actions, if anything is selected.